src/solver/model_getters.jl

```diff
@@ -237,6 +239,9 @@ function get_radiation_mode(parsed_args, ::Type{FT}) where {FT}
         "TRMM_LBA",
         "ISDAC",
     )
+    if !(radiation_name in ("allsky", "allskywithclear")) && reset_rng_seed
+        @warn "reset_rng_seed does not have any effect with " * radiation_name "radiation option"
```

Comment:
There is a missing `*` between `radiation_name` and `radiation_option`. 

I think the more Julian way would be to use string interpolation instead of concatenation:
```suggestion
        @warn "reset_rng_seed does not have any effect with rad = $(radiation_name)"
```

Reply:
There is no actual difference here!